canonical-web-and-design/ubuntu.com
# Update notification pattern for vanilla 3.0 in k8s docs

## Done

- The sequel to: https://github.com/canonical-web-and-design/ubuntu.com/pull/11310
- The notification pattern was updated, and support for the old version of the pattern is removed in Vanilla 3.x, this PR updates many instances across the project to the new structure, mostly within the k8s docs - see the upgrade guide: https://vanillaframework.io/docs/migration-guide-to-v3#notifications
- Note: running a find and replace on a couple of the classnames automatically linted some of these files, adding extra noise to the review...

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11314.demos.haus/
- Check the files that were changed, and find the pages on the demo, see that the notifications on them are not broken

## Issue / Card

(partially) fixes https://github.com/canonical-web-and-design/web-squad/issues/4854
